src/Step/Build/BackupData/WritableDirectories.php

```diff
@@ -107,35 +110,40 @@ public function execute()
             $this->backupDir($originalDir, $initDir);
         }
 
-        try {
-            $this->backupLogDir($magentoRoot . $logDir, $rootInitDir . $logDir);
-        } catch (\Exception $exception) {
-            throw new StepException($exception->getMessage(), $exception->getCode(), $exception);
-        }
+        $this->backupLogDir($magentoRoot . $logDir, $rootInitDir . $logDir);
```

Comment:
why do you use concatenation instead of $this->directoryList->getLog() ?

Reply:
I didn't add this line, it was moved from line 111

Reply:
"the moped is not mine..." :-p